paystack/src/main/java/co/paystack/android/Transaction.java

```diff
@@ -6,6 +6,8 @@ public class Transaction {
     private String id;
     private String reference;
 
+    public static Transaction NO_TRANSACTION = new Transaction();
```

Comment:
Minor, but personally I think that something like `EMPTY_TRANSACTION` is better naming.